scripts/30-globals.js.sh

```diff
@@ -1,3 +1,5 @@
 cat > /usr/share/nginx/dajsz/globals.js <<EOF
 window.tracking = "$GA_TRACKING_ID"
+window.apiUrl = "$API_URL"
+window.wsUrl = "$WS_URL"
```

Comment:
```suggestion
window.apiUrl = "${API_URL:-https://api.dajsz.hu}"
window.wsUrl = "${WS_URL:-wss://api.dajsz.hu}"
```
not 100% sure that this was the syntax but this will give a fallback value if the env vars are not defined